app/configurations/config.okc.js

```diff
@@ -68,6 +68,16 @@ export default configMerger(walttiConfig, {
   showVehiclesOnSummaryPage: true,
   showWeatherInformation: false,
 
+  defaultSettings: {
+    bikeSpeed: 5.59,
+    showBikeAndParkItineraries: false,
+  },
+
+  defaultOptions: {
```

Comment:
I think it is a bit dangerous that we implicitly assume throughout the code base that

- the config values are in the unit (e.g. land miles) that is widely used in the config's region;
- the users only ever want *one* unit, which disregards their browser's locale configuration.

But I can see that changing the code to support a "values with units" style is out of scope in this PR.